front/app/containers/SearchPage/SearchPage.tsx

```diff
@@ -503,7 +503,7 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
 
   componentDidMount() {
     let searchTerm = new URLSearchParams(this.props.location.search).getAll("q")
-    if (this.props.location.search) {
+    if (URLSearchParams(this.props.location.search).has("q")) {
```

Comment:
Did you test this?

Reply:
I think there may have been some confusion with my previous comment.  I mean you should use the has() function on the URLSearchParams instance.  I think what you have written is invalid since you aren't using the new keyword.  In either case it would be better to re-use the URLSearchParams you create on line 505 instead of creating a new one if that was your intention.

Reply:
I did test but I didn't push my local changes instead I used the GUI to just make the change to the one line. I was deep in some other changes on my local branch I didn't want to stash and switch gears. Seems like I missed the "new" in line 506. 

But I totally understand what you are saying. Sorry for the repeated back and forth, but think this next one should do it!